versions.tf

```diff
@@ -1,4 +1,11 @@
-
 terraform {
-  required_version = ">= 0.12"
-}
+  required_version = ">= 1.1"
+
+  required_providers {
+    aws = {
+      version = ">= 3.63"
+      source  = "hashicorp/aws"
+      version = ">= 4.9"
```

Comment:
Did this actually pass with two versions?

Reply:
Yeah because it passes the conditions, but I deleted the old one